src/Event/Value/Test/TestDox.php

```diff
@@ -18,24 +20,25 @@
 {
     private string $prettifiedClassName;
     private string $prettifiedMethodName;
-    private string $prettifiedAndColorizedMethodName;
 
-    public function __construct(string $prettifiedClassName, string $prettifiedMethodName, string $prettifiedAndColorizedMethodName)
+    public function __construct(string $prettifiedClassName, string $prettifiedMethodName)
     {
-        $this->prettifiedClassName              = $prettifiedClassName;
-        $this->prettifiedMethodName             = $prettifiedMethodName;
-        $this->prettifiedAndColorizedMethodName = $prettifiedAndColorizedMethodName;
+        $this->prettifiedClassName  = $prettifiedClassName;
+        $this->prettifiedMethodName = $prettifiedMethodName;
     }
 
     public function prettifiedClassName(): string
     {
         return $this->prettifiedClassName;
     }
 
+    /**
+     * @throws RuntimeException
+     */
     public function prettifiedMethodName(bool $colorize = false): string
     {
         if ($colorize) {
-            return $this->prettifiedAndColorizedMethodName;
+            throw new RuntimeException('Pretty printing method names is no longer supported for performance reasons.');
```

Comment:
This message is wrong: "pretty printing" still works, what no longer works is "colorized pretty printing".